build/package.proj

```diff
@@ -132,6 +133,12 @@
 		<Copy SourceFiles="@(PluginFiles)" DestinationFiles="@(PluginFiles->'$(BuildNuGetDir)\Content\%(RecursiveDir)%(Filename)%(Extension)')" />
 		<Copy SourceFiles="@(SrcFiles)" DestinationFiles="@(SrcFiles->'$(BuildNuGetDir)\src\%(RecursiveDir)%(Filename)%(Extension)')" />
 		<Copy SourceFiles="@(NuSpecFile)" DestinationFolder="$(BuildNuGetDir)" />
+
+		<!-- Umbraco -->
+    	<Copy SourceFiles="@(DashboardConfigXDTInstallFile)" DestinationFolder="$(BuildUmbDir)\App_Plugins\BulkUserAdmin\Install" />
```

Comment:
@jamiepollock Do we need the `*.xdt` in the Umbraco package file?
I thought it would only be for the NuGet package?